src/App.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./App.css";
+import Header from "./Header";
+import HeroesGrid from "./HeroesGrid";
+
+class App extends React.Component {
+  state = {
+    heroes: []
+  };
+
+  componentDidMount() {
+    this.getRequestInJSON("https://api.opendota.com/api/heroStats").then(
+      heroesGot => {
+        this.setState({
+          heroes: heroesGot,
```

Comment:
This name is kind of confusing, i think `displayedHeroes` would be better

Reply:
Are you confused with `heroesGot` or `heroes`?
Instead of `heoresGot`, just `heroes` would be enough in my opinion, but I'd extract `this.getRequestInJSON("https://api.opendota.com/api/heroStats")` to a `getHeroes()` function, so it would be extra clear that the response from this request is a hero

Reply:
I didn't use `heroes` just because I didn't like `heroes: heroes`, but I will change it to `heroes`. I think  `displayedHeroes` isn't suitable because at this part of the code, the heroes isn't been showed yet

Reply:
you could use the shorthand notation:
```
const name = 'saulo';
const objectve = {
  name,
}
```